all-spark-notebook/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM jupyter/all-spark-notebook:cf57ef145dce
+
```

Comment:
Can we get ssh/Git working?  Would the correct command be 

`apt-get install openssh-server openssh-client` ?

Reply:
Thanks @RobinL 

I think just the client is enough as we probably cannot or should not run `sshd` in the container